.github/workflows/main.yml

```diff
@@ -9,12 +9,13 @@ on:
 
 jobs:
   build:
+    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
```

Comment:
Why not just change the on pull-request to only be on: push to branch master?

Reply:
Because will make a tag with direct commits to master if a PR is bypassed

Reply:
Then the ```ref: ${{ github.ref_name }}``` can also be removed as well.

Reply:
There's branch protection so you can't push directly to master.